README.md

```diff
@@ -7,6 +7,33 @@
 
 > The TypeScript Definition Manager.
 
+## Notice Regarding TypeScript@2.0
+
+**For users doing `typings install dt~<package> --global` and receiving error**
+
+Starting from `TypeScript@2.0`, TypeScript user can acquire typings through `npm install @types/<package>`.
+
+Those typings are coming from the [`type-2.0`](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/types-2.0) branch of [DefinitlyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped).
```

Comment:
Not anymore though, since https://github.com/typings/typings/issues/738 (one month ago).